### DISCUSSION TITLE
Does this proxy servlet limit in any way the maximum size of a GET request?

### DISCUSSION THREAD

**Discussion by pgoldweic** [mitre/HTTP-Proxy-Servlet#255] [Q&A] [answered]
I'm having an issue with  a GET request that goes via the proxy servlet as being 'too long' (http error 413). Is there any GET request size limitation at all in the Proxy servlet itself? (I've already re-configured the app server - that hosts the initiator app and also the destination app- to allow a larger request size than usual to see if this resolves the issue, but I've had no success!)

**Comment by dsmiley** [comment, marked as answer]
If there is such a limit, then it'd be inside Apache HttpClient, which is what this proxy is based on.  Of course there might be a limit in your servlet environment too.

**Comment by pgoldweic** [comment]
Thanks for confirming this!

-Patricia

Patricia Goldweic
***@***.***
